Meadowlark-js/packages/meadowlark-utilities/src/Logger.ts

```diff
@@ -104,6 +104,15 @@ export function initializeLogging(): void {
       }),
     ],
   });
+
+  if (Config.get('SAVE_LOG_TO_FILE') === true) {
+    logger.add(
+      new winston.transports.File({
+        dirname: Config.get('LOG_FILE_LOCATION'),
```

Comment:
I noticed that the default value ends in a slash. What happens if we override with `LOG_FILE_LOCATION=/var/log` without the slash?

Reply:
I haven't tried that. I will do it

Reply:
There are no differences when overriding without the slash